src/Tests/Database/Group/GroupFixture.php

```diff
@@ -30,14 +30,20 @@ public function load(ObjectManager $manager): void
      */
     public static function generate(): TestData
     {
+        $parent = null;
+
         return TestData::from(new Group())
-            ->with('name', 'testing-web')
-            ->with('description', 'that one group to test kiwi')
+            ->with('name', 'test 1', 'test 2')
+            ->with('description', 'testgroup')
```

Comment:
"we've already implemented Alice in dev" is a stretch: it's installed and a number of fixtures are implemented, but it's nowhere used yet. I feel a change from the custom fixtures to alice is not within the scope of this PR.

Regarding the content changes here, @DavidckPixel any thoughts on this?

Reply:
While it might be a bit of a stretch so say it's fully implemented. We have merged it with the intend to use it. While changing all tests might be outside the scope of this PR. I would assume that the new tests do use Alice, writing them using "outdated" fixtures will only lead to more (useless) work.

Reply:
Given that there's currently not a single test using it and no other reference code on how to use Alice included anywhere within this repository, I feel that it's unreasonable to ask anyone at this point to use it, as they'd have to figure out for themselves how to get Alice to work within tests. The integration of Alice within kiwi is too unfinished that the lack of integration within this PR should be a blocking issue for merging it, imo.

Reply:
I felt the other comment was a better place for the conversation, since I do agree that changing to Alice if you're changing part of a test might be a lot to ask.

My suggestion would be to close this change as resolved (for the changing to Alice part. still curious as to what @DavidckPixel has to say), and to try to use Alice in all newly written tests.

Is that something you can agree with @jasperweyne?

Reply:
I'd like to see a dedicated PR first, that (at least partially) integrates Alice within a number of tests before we ask devs to switch over. That way, there exists some example code within the codebase when we ask devs to start using it.

Reply:
Can't say I completely agree, but I might be a bit biased since I was the one who introduced it. If you wanted at least some integration before expecting dev's to follow it, I feel that it should have been a requirement in the PR introducing Alice.

Before we derail this PR more then it already has. Lets agree to just leave it for this PR. I'll try and make a PR soon changing all our tests over and ripping the fixtures out.